online-store/src/components/data/data.ts

```diff
@@ -0,0 +1,225 @@
+export interface IItem {
```

Comment:
This date potentially comes from backed, so I would rather put it into json file or even use something like this https://jsonplaceholder.typicode.com/